src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Match.cs

```diff
@@ -80,9 +67,11 @@ internal Match(Regex regex, int capcount, string text, int begpos, int len, int
                                             "The parameters are out of range.");
         }
 
-        /*
-         * Nonpublic set-text method
-         */
+        /// <summary>
+        /// Returns an empty Match object.
+        /// </summary>
+        public static Match Empty => new Match(null, 1, string.Empty, 0, 0, 0);
```

Comment:
I think this is a mistake. Previously it would only make a singleton empty match object, now it's making one every time it's requested.

Reply:
Happy to see a test is failing for this reason: NextMatch_EmptyMatch_ReturnsEmptyMatch

Reply:
To speed things up I pushed this change.

Reply:
Correction, I didn't as I don't have permissions. It does fix the test though.

Reply:
Right, this need to be `{ get; } =` instead of `=>